x-pack/plugin/esql/qa/testFixtures/src/main/resources/floats.csv-spec

```diff
@@ -11,32 +11,32 @@ inDouble
 from employees | keep emp_no, height, height.float, height.half_float, height.scaled_float | where height in (2.03, 2.0299999713897705, 2.029296875, 2.0300000000000002) | sort emp_no;
 
 emp_no:integer |height:double  |height.float:double |height.half_float:double |height.scaled_float:double
-10001          |2.03           |2.0299999713897705  |2.029296875              |2.0300000000000002
-10090          |2.03           |2.0299999713897705  |2.029296875              |2.0300000000000002
+10001          |2.03           |2.0299999713897705  |2.029296875              |2.03
```

Comment:
You need to add a new capability for this fix and use it in this test similar to how `fix_parsing_large_negative_numbers` is used above. This is needed to fix backward compatibility tests.